browser/extensions/api/moonpay_api.cc

```diff
@@ -0,0 +1,91 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/extensions/api/moonpay_api.h"
+
+#include <string>
+#include <memory>
+#include <utility>
+
+#include "brave/browser/profiles/profile_util.h"
+#include "brave/components/moonpay/browser/regions.h"
+#include "brave/components/moonpay/common/pref_names.h"
+#include "brave/components/ntp_widget_utils/browser/ntp_widget_utils_region.h"
+#include "chrome/browser/profiles/profile.h"
+#include "components/prefs/pref_service.h"
+
+namespace {
+  bool IsMoonpayAPIAvailable(content::BrowserContext* context) {
+    Profile* profile = Profile::FromBrowserContext(context);
+    return profile->IsRegularProfile();
```

Comment:
@ryanbr It is not using our `brave::IsRegularProfile` in `browser/profiles/profile_util.cc` as @bbondy suggested above, is this intentional? I believe what you want here should be `brave::IsRegularProfile`.

Reply:
fixed